packages/core/src/routes/account/mfa-verifications.ts

```diff
@@ -135,4 +135,47 @@ export default function mfaVerificationsRoutes<T extends UserRouter>(
       return next();
     }
   );
+
+  router.delete(
+    `${accountApiPrefix}/mfa-verifications/:verificationId`,
+    koaGuard({
+      params: z.object({
+        verificationId: z.string(),
+      }),
+      status: [204, 400, 401],
+    }),
+    async (ctx, next) => {
+      const { id: userId, scopes, identityVerified } = ctx.auth;
+      assertThat(
+        identityVerified,
+        new RequestError({ code: 'verification_record.permission_denied', status: 401 })
+      );
+      const { fields } = ctx.accountCenter;
+      assertThat(
+        fields.mfa === AccountCenterControlValue.Edit,
+        'account_center.filed_not_editable'
+      );
+      assertThat(scopes.has(UserScope.Identities), 'auth.unauthorized');
+
+      const user = await findUserById(userId);
+      const mfaVerification = user.mfaVerifications.find(
+        (mfaVerification) =>
+          mfaVerification.id === ctx.guard.params.verificationId &&
+          mfaVerification.type === MfaFactor.WebAuthn
```

Comment:
According to your API naming convention, should it not limit the verification type to WebAuthn?  Can any type of mfa-verificaiton should can be deleted by a given ID?